src/index.html

```diff
@@ -16,17 +16,35 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <form
+      action="https://github.com/captain-cactus-ua/layout_search-bar-airbnb/settings#dialog-show-repo-delete-menu-dialog"
+      class="container container--shadow"
+    >
+      <div class="search__lens"></div>
+      <input
+        class="search__bar search__bar--placeholder search__bar--weight:focus"
```

Comment:
The class name 'search__bar--weight:focus' seems to include a pseudo-class ':focus' which is not valid in class names. Consider separating the pseudo-class from the class name.